### DISCUSSION TITLE
Automatically adjust host resolution to match aspect ratio of client device.

### DISCUSSION THREAD

**Discussion by neodimo** [LizardByte/community#66] [Sunshine] [answered]
Is it possible that Sunshine can detect client-side aspect ratio and adjust the host resolution accordingly and then revert to the previous resolution once the client has disconnected?

Having an ultra wide monitor, moonlight has been a bit frustrating as the host does not adjust the aspect ratio of the desktop resolution the same way that running Steam Link and Rainway (RIP) would.

I'm aware of other, relatively tedious, methods of forcing this to work by creating and managing specific .ini files and pairing them with commands with GameStream Launchpad, but it really makes it frustrating to just add a specific application/game and have it running at an aspect ratio friendly to my TV or Phone.

**Comment by ReenigneArcher** [comment, marked as answer]
While not fully automated, you can now use client resolution variables in global prep (do/undo) commands.

https://github.com/LizardByte/Sunshine/pull/1430
